#! /usr/bin/env python

from dynamic_reconfigure.parameter_generator_catkin import *


PACKAGE = 'jsk_arc2017_common'
ID = 'PublishDatasetV3'

gen = ParameterGenerator()

gen.add('scene_idx', int_t, 0, 'Index of scene.', 0, 0, 10000)
gen.add('frame_idx', int_t, 0, 'Index of view frame in scene.', 0, 0, 8)

exit(gen.generate(PACKAGE, PACKAGE, ID))
